### PR TITLE
Update spellCheck.py

### PR DESCRIPTION
**Pull request by GadeManasa**
Special characters are removed from the start and end of the word

### CODE REVIEW

File: Flask-Integration/spellCheck.py [hunk 1]
```diff
@@ -60,6 +60,8 @@ def checkPhrase(phrase, lng):
     
     #print(textList)
     for word in textList: 
+        word = re.sub('^[^a-zA-Záéíóú]','',word) 
+        word = re.sub('[^a-zA-Záéíóú]*$','',word)
```
Comment (rjuliang) [inline Flask-Integration/spellCheck.py:64]: Hi Manasa, 
Is there a reason the sub function is repeated?
Thanks,
Julian

Comment (GadeManasa) [reply Flask-Integration/spellCheck.py:64]: Hi Julian ,
I tried to use  or operator in sub function but  it didn't work accordingly. As a result, I had to perform it twice to remove special characters from the beginning and end.

Comment (rjuliang) [reply Flask-Integration/spellCheck.py:64]: Perfect!
Thank you Manasan

**Review by rjuliang — APPROVED** [review]
After verifying with Manasa, these changes look good to go.

**Pull request merged**